### ISSUE TITLE
v16.3.17 CRUD Operation ERROR

### ISSUE THREAD

**Issue by arifnurdiansyah92** [syncfusion/ej2-javascript-ui-controls#30]
After update to this version, crud operation all fails, the problem is the data can't bind with the model (becoming null)

NOTE : Working fine in previous version

**Comment by Madhust** [comment]
Hi @arifnurdiansyah92 ,

Could you please update the below information regarding your problem?

- Please mention the control name whether you are using grid or schedule.
- Please share the data adapter used by your control.
- Please share the code example or screenshot to replicate the problem.

The above information would help us to analyze and provide the solution as earlier as possible.

Madhu

**Comment by jvhariharan** [comment]
Hi @arifnurdiansyah92,

Thanks for your update.

We have validated the provided information and checked the reported problem with our end unfortunately we are unbale to reproduce it. We have prepared a sample for your reference. In the below sample, we have performed the CRUD operation with WebApiAdaptor and its working fine at our sample.

Kindly refer to the below sample for more information.

Sample : http://www.syncfusion.com/downloads/support/directtrac/general/ze/EJ2-grid-crud-webapi-1393988194 

<img width="548" alt="dhi" src="https://user-images.githubusercontent.com/26004767/45541357-84bf4700-b82c-11e8-81c5-84a95d3a1bdf.png">


1.	In your code example you are applied visible property false for the primary key column so you are generating the primary key value in server side?
2.	The reported problem occurred only for POST action or for both PUT & POST?
3.	If possible reproduce the reported problem with our sample or share the demo sample.  

Regards,
Hariharan
